quicktests/overlaying/tests/functional/formatter.js

```diff
@@ -12,7 +12,17 @@ function run(svg, data, Plottable) {
     };
 
 
-  var big_numbers = data[0].slice(0, 5);
+
+  var deep_copy = function(from, to){
```

Comment:
Probably want to rename to deepCopyPoints and then wrap to a util like makeRandomData?